### PR TITLE
Bugfix/update docker image coordinates

### PR DESCRIPTION
**Pull request by fmisir**
This PR fixes two issues:

1. Hard coded docker image coordinates in the docker-compose file for catena-x@home
2. Add the default *.pfx file so that the demo works out of the box. Without that the control plane containers do not start.

### CODE REVIEW

File: eclipsedataspaceconnector/api-wrapper/.gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ build/
 adapter/
 semantic-hub/
 registry/
+!*.pfx
```
Comment (florianrusch-zf) [inline eclipsedataspaceconnector/api-wrapper/.gitignore:7]: Please add an empty line at the end

```suggestion
!*.pfx

```

**Review by florianrusch-zf — APPROVED** [review]
I'm not really a fan of having this env variable in the docker-compose file, because now you can't anymore use docker-compose directly...but I also get you point why you want to do it :-(

I think if we want to use catena-x@home for a longer period we should think of an other solution. But for now it's ok for me

**Comment by fmisir** [comment]
> I'm not really a fan of having this env variable in the docker-compose file, because now you can't anymore use docker-compose directly...but I also get you point why you want to do it :-(
> 
> I think if we want to use catena-x@home for a longer period we should think of an other solution. But for now it's ok for me

Good point. We could introduce a `.env` - with a defaults. As it will be picked up by default by docker-compose, so that you could still use docker-compose.

I'am just wondering if we really need to support two docker registries. I would prefer to just stick to `ghcr.io` at least for Catenax@Home.

**Comment by florianrusch-zf** [comment]
> I'am just wondering if we really need to support two docker registries. I would prefer to just stick to `ghcr.io` at least for Catenax@Home.

I'm totally with you :)

**Pull request closed without merging**